### ISSUE TITLE
Embed: Can't focus Embed block when empty

### ISSUE THREAD

**Issue by mcsf** [WordPress/gutenberg#29374]
## Description

In certain cases, the Embed block may fail to embed a source and not realise that it has failed (https://github.com/WordPress/gutenberg/issues/29429). When this happens, the Embed block is rendered as an invisible rectangle that can't be focused with the cursor.

## Step-by-step reproduction instructions
1. In an empty post, paste `https://woocommerce.com/experts/neuralab/`.
2. The console should reveal the following failure: `Refused to display 'https://woo...' in a frame because it set 'X-Frame-Options' to 'SAMEORIGIN'.`.
3. An Embed block "appears" as an invisible rectangle.
4. Try unselecting the block and re-selecting it using the cursor. **No selection occurs.**

## Screenshots or screen recording (optional)

<img width="746" alt="Captura de ecrã 2021-02-26, às 12 38 17" src="https://user-images.githubusercontent.com/150562/109301444-e2df1980-782f-11eb-8d7a-91c1ce905090.png">

**Comment by mcsf** [comment]
@ellatrix: ^ in case this rings any bells

**Comment by ellatrix** [comment]
Interesting @mcsf, it also refuses to inspect the element. Will have a quick look.

**Comment by ellatrix** [comment]
Hm that's strange. I pasted the URL in the demo page, clicked next to the block to deselect, and then clicked back on the embed block and it works in that scenario. Trying just that URL on an empty page fails.

**Comment by ellatrix** [comment]
@mcsf Strange, I tested again and it works fine...

![embed-select](https://user-images.githubusercontent.com/4710635/111176055-ab9f9500-85b1-11eb-9c5f-984e128d1f4e.gif)
